storage/LegacyStorageQuickstart/StorageExample.xcodeproj/project.pbxproj

```diff
@@ -66,6 +70,8 @@
 		10D187C11C8640E300D0DC91 /* DownloadViewController.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = DownloadViewController.h; sourceTree = "<group>"; };
 		10D187C21C8640E300D0DC91 /* DownloadViewController.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = DownloadViewController.m; sourceTree = "<group>"; };
 		10D187C41C87608900D0DC91 /* DownloadViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = DownloadViewController.swift; sourceTree = "<group>"; };
+		308175C3CBFD1759BE0E835C /* Pods-StorageExampleTests.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-StorageExampleTests.release.xcconfig"; path = "Target Support Files/Pods-StorageExampleTests/Pods-StorageExampleTests.release.xcconfig"; sourceTree = "<group>"; };
+		42244572AEEC8FBEE20ED203 /* Pods-StorageExample.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-StorageExample.release.xcconfig"; path = "Target Support Files/Pods-StorageExample/Pods-StorageExample.release.xcconfig"; sourceTree = "<group>"; };
```

Comment:
nit: run `pod deintegrate`

Reply:
This is for the `LegacyStorageQuickstart` which should have the `Pods` if I understand correctly. I have `pod deintegrate` the `StorageExample.xcodeproj` under  the `storage/StorageExample/` dir.

Reply:
Will merge tomorrow to avoid SDK repo CI failures caused by the the new app.